surefire-its/pom.xml

```diff
@@ -241,6 +262,27 @@
                             </execution>
                         </executions>
                     </plugin>
+                  <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-invoker-plugin</artifactId>
+                    <configuration>
+                      <localRepositoryPath>${settings.localRepository}</localRepositoryPath>
+                      <extraArtifacts>
+                        <extraArtifact>org.apache.maven.plugins:maven-surefire-report-plugin:${project.version}</extraArtifact>
+                        <extraArtifact>org.apache.maven.plugins:maven-surefire-plugin:${project.version}</extraArtifact>
+                        <extraArtifact>org.apache.maven.plugins:maven-failsafe-plugin:${project.version}</extraArtifact>
+                      </extraArtifacts>
```

Comment:
invoker:install also install all project dependencies with transitive with scope compile + runtime to local-repo, 
so we can skip extraArtifacts if change scope to runtime of those